forgettable-frontend/src/components/CustomButton/CustomButton.module.css

```diff
@@ -21,7 +21,7 @@
 }
 
 .Text{
-    color: black;
+    color: var(--txt1);
     font-size: 18px;
```

Comment:
Are we going to use global CSS for the font size here or keep it hard-coded?

Reply:
ahh sorry missed that! will do another fix!